_ml-commons-plugin/remote-models/index.md

```diff
@@ -218,7 +218,7 @@ When the operation is complete, the state changes to `COMPLETED`:
 
 Take note of the returned `model_id` because you’ll need it to deploy the model.
```

Comment:
Should we highlight this key info in a callout?

Reply:
I don't use a callout in other places where there are similar steps... I think it's because of the proximity of the next step, which requires the model ID. Users can also retrieve the model ID after the fact if they don't note it immediately.